grails-app/migrations/install/changelog-create-schema.groovy

```diff
@@ -3049,10 +3065,44 @@ databaseChangeLog = {
             column(name: "source_type", type: "VARCHAR(255)")
 
             column(name: "replenishment_type_code", type: "VARCHAR(255)")
+
+            column(name: "approved_by_id", type: "CHAR(38)")
+
+            column(name: "date_approved", type: "datetime")
```

Comment:
Capitalize the DATETIME.

Reply:
Sure, will do it (but I'll need to adjust it in the entire file, because it is lower in every case of `datetime`)

Reply:
So that was auto-generated as well?

Reply:
Yup